hz: 50

dt: 0.1
predict_time: 4.0

goal_tolerance: 0.5

max_vel: 0.45
min_vel: 0.0
max_yawrate: 1.0
min_yawrate: -1.0
max_accel: 1000.0
max_dyawrate: 1000.0

v_reso: 0.05
y_reso: 0.1

heading_cost_gain: 1.0
velocity_cost_gain: 1.0
distance_cost_gain: 1.0

robot_radius: 0.25
radius_margin: 0.1

search_range: 0.95

robot_frame: "base_link"